docs/learn/learn-guides-polkadot-opengov.md

```diff
@@ -65,3 +65,25 @@ within the track, and a
 have been met. Failing to submit the decision deposit within a
 {{ polkadot: <RPC network="polkadot" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :kusama }}-day
 period will lead to a referendum timeout.
+
+## Request Submission and Decision Deposit Back
+
+If you submitted a proposal and a referendum for such proposal has ended, you can claim your
+submission and decision deposits back by issuing the `refundSubmissionDeposit` and the
+`refundDecisionDeposit` extrinsics, respectively. Those calls will succeed if the proposal is
+completed as passing or failing.
+
+Users can not refund their submission deposit while the referendum is `Ongoing`. Similarly, users
+cannot refund their submission deposit if the proposal has `TimedOut` (failing to submit the
+decision deposit within a 14-day period will lead to a referendum timeout). This behavior exists so
+that users can refrain from spamming the chain with proposals that have no interest from the
+community. If a proposal is in the `TimedOut` state, any user can call `slash_proposal_deposit`,
+which will move the funds from the user to a runtime-configured account, like the treasury.
+
+If you submitted a proposal and no decision deposit has been placed on that referenda, the
+submission deposit will not be claimable using the `refundSubmissionDeposit` extrinsic. To refund
+your deposit, you can start a new referendum and specifically request a refund from the treasury.
+You need to make sure you have enough balance for a new submission deposit and you will need to
+select the right track to ask for the refund. In any case, the
+[Small Tipper Track](../maintain/maintain-guides-polkadot-opengov.md#small-tipper) would be fine for
+any kind of deposit.
```

Comment:
```suggestion
Users can not refund their submission deposit while the referendum is `Ongoing`. Similarly, users
cannot refund their submission deposit if the proposal has `TimedOut` (failing to submit the
decision deposit within a 14-day period will lead to a referendum timeout). This behavior exists so
that users can refrain from spamming the chain with proposals that have no interest from the
community. If a proposal is in the `TimedOut` state, any user can call `slash_proposal_deposit`,
which will move the funds from the user to a runtime-configured account, like the treasury.

To refund your slashed deposit, you can start a new referendum and specifically request a refund from the treasury.
You need to make sure you have enough balance for a new submission and decision deposit, and you will need to
select the right track to ask for a refund. For example, the
[Small Tipper Track](../maintain/maintain-guides-polkadot-opengov.md#small-tipper) would be fine for submission deposit on all tracks.
```

Reply:
implemented, added onchain fetch of timeout for polkadot and kusama